main.tf

```diff
@@ -76,9 +76,9 @@ resource "aws_backup_plan" "default" {
       }
 
       dynamic "copy_action" {
-        for_each = lookup(rule.value, "destination_vault_arn", null) != null ? ["true"] : []
+        for_each = toset(lookup(rule.value, "destination_vault_arns", []))
```

Comment:
Could you update the examples/complete to give it multiple destination arns to make sure it works?

Reply:
yep sure. will do